homeassistant/helpers/intent.py

```diff
@@ -386,7 +387,10 @@ async def async_handle(self, intent_obj: Intent) -> IntentResponse:
         )
 
         if not states:
-            raise IntentHandleError("No entities matched")
+            raise IntentHandleError(
+                "No entities matched for: " f"name={name}, " f"area={area}, ",
```

Comment:
Weird formatting here.

Reply:
This was just to avoid a really long format string. I'll change it back to normal.